qa/workunits/rados/test_rados_tool.sh

```diff
@@ -316,9 +316,11 @@ test_xattr() {
     $RADOS_TOOL -p $POOL listxattr $OBJ > $V1
     grep -q foo $V1
     grep -q bar $V1
-    wc -l $V1 | grep -q "^2 "
+    cat $V1
+    [ `cat $V1 | wc -l ` -eq 2 ]
     rm $V1 $V2
     cleanup
+    echo test_xattr done
```

Comment:
could you please remove it? i'd like to keep this commit as a refactor only change.